options.html

```diff
@@ -99,6 +99,11 @@ <h2>
                 <div id="video_loader_settings">
                     <input type="checkbox" id="video_loader_hd"/><label for="video_loader_hd">Show in HD (when available)</label>
                 </div>
+                <p>
+                    <input type="checkbox" class="script_check" id="embed_socials" />
+                    <label for="video_loader">Embed Social Links</label>
```

Comment:
Wrong label target here; if you click the "Embed Social Links" text it checks "Embed videos" instead.

Reply:
Ahhh, good catch.